docs/rules/no-useless-scu.md

```diff
@@ -0,0 +1,64 @@
+# Prevent usage of shouldComponentUpdate when extending React.PureComponent (react/no-useless-scu)
+
+Warns if you have `shouldComponentUpdate` defined when defining a component that extends React.PureComponent.
+While having `shouldComponentUpdate` will still work, it becomes pointless to extend PureComponent.
+
+## Rule Details
+
+The following patterns are considered warnings:
+
+```jsx
+class Foo extends React.PureComponent {
+  shouldComponentUpdate() {
+    return true;
```

Comment:
is the "return true" important? if not, i'd maybe put a comment here or something to indicate "any code can go here" (same throughout)